redisearch/client.py

```diff
@@ -501,19 +502,20 @@ def info(self):
         it = six.moves.map(to_string, res)
         return dict(six.moves.zip(it, it))
 
-    def _mk_query_args(self, query):
+    def _mk_query_args(self, query, query_params):
         args = [self.index_name]
 
         if isinstance(query, six.string_types):
             # convert the query from a text to a query object
             query = Query(query)
         if not isinstance(query, Query):
             raise ValueError("Bad query type %s" % type(query))
-
         args += query.get_args()
+        if query_params is not None:
+            args+= Query.get_params_args(query_params)
```

Comment:
Do you want to use the member `query._params` if argument `query_params` is `None`?
(or even to merge both if both are not `None`, giving precedence to the argument over the member?)
Or do you want to get rid of the member? (it is only available for `Query`, not for `AggregateRequest`)